src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import SingleInput from '../components/SingleInput';
+
+class LoginFormContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {username: '', password: ''};
+
+        this.handleEvent = this.handleEvent.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.clearForm = this.clearForm.bind(this);
+        this.handleUsernameCheck = this.handleUsernameCheck.bind(this);
+    }
+
+    clearForm() {
+        this.setState({username : '', password: ''});
+    }
+
+    handleEvent(event) {
+        const init = {
+            method: 'post',
+            headers: {
+                "Content-type": 'application/json',
+            },
+            body: JSON.stringify({
+                username: this.state.username,
+                password: this.state.password,
+            }),
+        };
+
+        fetch(`/${event.target.name}`,init).then((res) => {
+            if (res.ok) {
+                console.log('post success');
+            } else {
+                console.log('post unsuccess');
+            }
+        });
+
+        this.clearForm();
+
+        event.preventDefault();
+    }
+
+    handleChange(event) {
+        this.setState({[event.target.name]: event.target.value});
+    }
+
+    handleUsernameCheck(event) {
+        const init = {
+            method: 'post',
+            headers: {
+                "Content-type": 'application/json',
+            },
+            body: JSON.stringify({
+                username: this.state.username,
+            }),
+        };
+
+        fetch('/checkUsername',init).then((res) => {
+            res.json().then((json) => {
+                if (!json.exists) {
+                    alert('Username available');
```

Comment:
Please don't use alert in your final version.  It's so incredibly jarring and makes me want to destroy things.